lib/danger/danger_core/plugins/dangerfile_bitbucket_server_plugin.rb

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+require 'danger/plugin_support/plugin'
+
+module Danger
+  class DangerfileBitbucketServerPlugin < Plugin
+    def self.new(dangerfile)
+      return nil if dangerfile.env.request_source.class != Danger::RequestSources::BitbucketServer
+      super
+    end
+
+    def self.instance_name
+      "bitbucket_server"
```

Comment:
I don't know the terminology (as I've not used BitBucket as a use) but I'm willing to bet that this should probably be `bitbucket`, e.g. I would expect to write `bitbucket.pr_json` instead of `bitbucket_server.pr_json`